page_objects/authenticationPage.js

```diff
@@ -1,17 +1,35 @@
 module.exports = {
     url: "http://automationpractice.com/index.php?controller=authentication&back=my-account",
```

Comment:
Lets imagine that we have different environments. Could you please create kind of property file and get url from there

Reply:
done